### PR TITLE
CB-9053. Cleanup and update diagnostics bundle format + start using CDP doctor (CB-8865)

### PR DESCRIPTION
**Pull request by oleewere**
details:
- refactor cloud storage related parameters a bit (also fixes some issue)
- start using newer cdp-telemetry cli tool (include multiple workers feature, but use only 1 for those at the moment)
- update bundle to include bundle-info.json that contains required data that can be used by EDH
- include filecollector-metadata.json that includes the processing results
- rename bundle_info.json to diagnostics_request.json as that contains the diagnostics request data
- license id is not filled that
- include some new fields on the API (altohgh many of that are generated but as we passes those requests to internal endpoints, include those fields in the request)

See detailed description in the commit message.

### CODE REVIEW

File: common-model/src/main/java/com/sequenceiq/common/model/diagnostics/CloudStorageDiagnosticsParameters.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.common.model.diagnostics;
+
+import java.util.Map;
+
+public interface CloudStorageDiagnosticsParameters {
+
+    /**
+     * Get cloud specific parameters ans a key value map for diagnostics
```
Comment (attilapalfi92) [inline common-model/src/main/java/com/sequenceiq/common/model/diagnostics/CloudStorageDiagnosticsParameters.java:8]: typo here "ans"

File: datalake/src/main/java/com/sequenceiq/datalake/service/sdx/diagnostics/DiagnosticsService.java [hunk 2]
```diff
@@ -30,6 +31,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DiagnosticsService.class);
 
+    @Value("${info.app.version:}")
+    private String version;
```
Comment (attilapalfi92) [inline datalake/src/main/java/com/sequenceiq/datalake/service/sdx/diagnostics/DiagnosticsService.java:35]: where is it used?

**Review by attilapalfi92 — COMMENTED** [review]
1 small typo and a question

File: datalake/src/main/java/com/sequenceiq/datalake/service/sdx/diagnostics/DiagnosticsService.java (hunk 2 above)
Comment (oleewere) [reply datalake/src/main/java/com/sequenceiq/datalake/service/sdx/diagnostics/DiagnosticsService.java:35]: it.is only used as a default if for some reason the blueprint for the cluster does not exist

Comment (attilapalfi92) [reply datalake/src/main/java/com/sequenceiq/datalake/service/sdx/diagnostics/DiagnosticsService.java:35]: but I cannot see it referenced anywhere

**Review by attilapalfi92 — APPROVED** [review]

**Review by attilapalfi92 — APPROVED** [review]

**Pull request merged**